lighthouse-core/test/results/sample_v2.json

```diff
@@ -1101,7 +1101,7 @@
       "scoreDisplayMode": "numeric",
       "name": "bootup-time",
       "description": "JavaScript boot-up time",
-      "helpText": "Consider reducing the time spent parsing, compiling, and executing JS. You may find delivering smaller JS payloads helps with this. [Learn more](https://developers.google.com/web/lighthouse/audits/bootup).",
+      "helpText": "Consider reducing the time spent parsing, compiling, and executing JS. You may find delivering smaller JS payloads helps with this. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/bootup).",
```

Comment:
@paulirish yeah ignore help text is a good call for our golden file assertions :)